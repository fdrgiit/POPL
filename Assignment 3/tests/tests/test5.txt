TYPE
   at : INT;
   bt : at;
   ct : dt;
   et : BOOLEAN;
   ft : STRING;
   gt : REAL;

 VAR  
   av : at;   bv : bt;   cv : ct;   ev : et;   fv : ft;   gv : gt;
   hv : t1;   iv : t1;   jv : t2;   kv : t2;   lv : t3;

 {
    av = 1;
    bv = av + lv;
    WHILE jv < iv
    {
       av = av+1;
    }

    WHILE jv
    {
       av = av+1;
    }

    bv = av;
    cv = 1.0;
    mv = nv;
 }

