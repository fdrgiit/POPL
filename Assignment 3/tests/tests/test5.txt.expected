INT : at bt t3 av bv lv #
REAL : ct gt dt cv gv #
STRING : ft fv #
BOOLEAN : et t1 t2 ev hv iv jv kv #
mv : nv #

